From: Reinhard Tarter <siretart@tauware.de>
Subject: Revert commit 0e1f67b725088c4d7425f56e260ddbb44078d11b
Original-Author: Giuseppe Scrivano <gscrivan@redhat.com>
Date:   Tue Oct 26 21:05:42 2021 +0200

    cgroups: use SessionBusPrivateNoAutoStartup
    
    do not start up a dbus daemon if it is not already running.
    
    [NO NEW TESTS NEEDED] the fix is in a dependency.
    
    Closes: https://github.com/containers/podman/issues/9727
    
    Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

--- a/pkg/cgroups/cgroups.go
+++ b/pkg/cgroups/cgroups.go
@@ -464,7 +464,7 @@
 // GetUserConnection returns a user connection to D-BUS
 func GetUserConnection(uid int) (*systemdDbus.Conn, error) {
 	return systemdDbus.NewConnection(func() (*dbus.Conn, error) {
-		return dbusAuthConnection(uid, dbus.SessionBusPrivateNoAutoStartup)
+		return dbusAuthConnection(uid, dbus.SessionBusPrivate)
 	})
 }
 
